src/components/modules/chat/server/chat.firebase.ts

```
@@ -0,0 +1,143 @@
+  attachment?: {
+    name: string;
+    type: string;
+    data: string;
+  };
+}
+
+export const getOrCreateChat = async (
+  userId: string,
+  contactId: string,
+): Promise<string> => {
+  const q = query(chatsCollection, where("participants", "array-contains", userId));
```

ESLint

[error] Replace `chatsCollection,·where("participants",·"array-contains",·userId)` with `⏎····chatsCollection,⏎····where("participants",·"array-contains",·userId),⏎··`

(prettier/prettier)